### DISCUSSION TITLE
where to route usb cable for pack

### DISCUSSION THREAD

**Discussion by ecto1a2003** [gpstar81/GPStar-proton-pack#240] [Q&A] [answered]
I have the usb connected. I cant seem top get the motherboard back on. do I remove the battery box? manual doesn't go over this.

**Comment by gpstar81** [comment]
Are you referring to the programming cable? You can route it through the channels in the Pack shell. The battery box can stay, though it is recommended to remove it as it takes up a good amount of space.

**Reply by ecto1a2003** [reply, marked as answer]
Ill probably doo that next time i have it open. I couldn't get the motherboard to fully close up eith the programming cable still in, i tried several routes.
